driverapi/driverapi.go

```diff
@@ -19,6 +19,8 @@ type UUID string
 
 // Driver is an interface that every plugin driver needs to implement.
 type Driver interface {
+	Config(config interface{}) error
```

Comment:
Looking at the changes in CreateNetwork() in bridge.go, seems like the function will no longer take the driver specific config and will return error if driver does not already have the config populated.

If this is the case, the following comment for CreateNetwork() should be updated, config is now probably network specific config.

To be consistent with the nice comments on the other methods in the interface, can you please add a comment on top of Config().